tests/GSuitePartnerClientTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Gsuite\V1\GetDomainInformationRequest;
+use PHPUnit\Framework\TestCase;
+use Vendasta\GSuite\V1\GSuitePartnerClient;
+
+class GSuitePartnerClientTest extends TestCase
+{
+    public function testGetDomainInformationHappyPath()
+    {
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $client = new GSuitePartnerClient($environment);
+
+        $req = new GetDomainInformationRequest();
+        $req->setDomain("google.com");
+
+        try {
+            $resp = $client->GetDomainInformation($req);
+        } catch (Vendasta\Vax\SDKException $e) {
+            self::fail('unexpected failure');
+            return;
+        }
+
+        self::assertNotEmpty(
+            $resp->getDomainInformation(),
```

Comment:
I'm assuming this test doesn't actually pass when you run it? We shouldn't have any domain information for google.com

Reply:
It doesn't pass, but I don't think any URL would pass generally as the users of the SDK wouldn't have access to account group that has any particular domain.

Reply:
Ya, we could assert that the response does a 404 or permission denied, but I'm okay either way since the tests are really just to give a code example